sr_port/anticipatory_freeze.h

```diff
@@ -101,7 +101,8 @@ error_def(ERR_TEXT);
 	{														\
```

Comment:
The below bolded commit message text is confusing. Can it be removed?

```- sr_unix/gtm_getpwuid.c -```
**Originally had changes for an issue but backed those out but left
                           these changes intact anyway.**
```
  - Removed an unneeded return value check from malloc (gtm_malloc does not return if
    no storage).
  - Reformatted an #ifdef block.
```

Reply:
Ok will do. It's just noting that there are no static-scan related issued "fixed" in this module - just standards stuff and removing unneeded code.

Reply:
Can the  interface change to CLEAR_ANTICIPATORY_FREEZE be mentioned in the commit message. I don't see it in the latest commit message.